sql/_13_issues/_24_1h/cases/cbrd_25098.sql

```diff
@@ -0,0 +1,89 @@
+-- Verified the CBRD-25098
+-- Problem where inst_num() is inappropriately changed to order_by_num() when a subquery containing an ORDER BY is view merged
+
+-- orderby_num()
+-- create table & insert data
+drop table if exists tbl;
+create table tbl(col1 int, col2 int, col3 int);
+insert into tbl values(1,1,1),(2,2,2),(3,3,3);
+
+-- const order by
+select
+	/*+ RECOMPILE */
+	col2,col3,orderby_num()
+	,(select 1 from tbl order by col1 limit 1) as "scala col1"
+from tbl 
+where col2 = 1 
+order by col2 for orderby_num() <= 1;
+
+-- const order by with group by
+select
+	/*+ RECOMPILE */
+	col2,col3,orderby_num()
+	,(select col2 from tbl order by col1 limit 1) as "scala col1"
+from tbl 
+where col1 = 1 
+group by col1,col2 
+order by col1;
+
+
+-- view merge
+-- create table & insert data
+drop table if exists tbl1;
+drop table if exists tbl2;
+
+create table tbl1 (col1 int, col2 int);
+create table tbl2 (col3 int, col4 int);
+
+insert into tbl1 values (1,1),(2,2),(3,3);
+insert into tbl2 values (1,1),(2,2),(3,3);
+
+
+-- view merge 1
```

Comment:
Add query by adding limit 1 to view merge 1, 2
select ... from (...)x limit 1;

Reply:
Okay I will reflect on that.